angularjs-portal-frame/src/main/webapp/css/buckyless/home.less

```diff
@@ -400,6 +400,18 @@ div.table-cell {
   width:100%;
   max-width:100%;
 }
+.header-opaque .second-description {
```

Comment:
Is this "second-description", or is this maybe "smart-widget-selector-message" ?  Seems like semantically this is less about being a second description and more about being the specific magic incantation required to get that pointer to float pointing at that favorite mode selector control.